chcemvediet/apps/anonymization/admin.py

```diff
@@ -0,0 +1,73 @@
+# vim: expandtab
```

Comment:
*Komentar sa tyka binarneho `anonymization.odt`, ku ktoremu sa nedaju davat komentare*

```
attachment: Object; May NOT be NULL.
```

Preco "Object"? Ved vieme, ze to je vzdy FK na "Attachment". A preco spominas "Polymorphic relation to Attachment database object." ked to je normalny FK?

Reply:
:ok:

Reply:
:ok: